package.json

```diff
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "@docusaurus/core": "^3.1.0",
-    "@ionic-docs/preset-classic": "^1.0.2",
+    "@ionic-docs/preset-classic": "^1.1.7",
```

Comment:
latest is 1.2.7

Shouldn't matter much since it has `^`, so should pick latest anyway